dev.tfkhdyt.pomodoro.yml

```diff
@@ -0,0 +1,42 @@
+id: dev.tfkhdyt.pomodoro
```

Comment:
https://tfkhdyt.dev/ seems not reachable.

Reply:
`Host tfkhdyt.dev not found: 3(NXDOMAIN)`

no such domain